nomad/fsm.go

```diff
@@ -1514,6 +1549,15 @@ func (n *nomadFSM) Restore(old io.ReadCloser) error {
 			if err := restore.CSIVolumeRestore(plugin); err != nil {
 				return err
 			}
+		case EventSinkSnapshot:
```

Comment:
We need a `persistEventSinks` function to go with this for the snapshot.

Reply:
woops, thanks